db/migrate/20160516070529_divide_lookup_key_permissions.rb

```diff
@@ -1,14 +1,32 @@
 class DivideLookupKeyPermissions < ActiveRecord::Migration
+  class FakeFilter < ActiveRecord::Base
```

Comment:
:+1:  didn't know it'd make any validation fail, I agree with using fake models in migrations